### PR TITLE
fix(components): add module entry point

### PR DESCRIPTION
**Pull request by jennieramida**
Fix CFE-547

### CODE REVIEW

File: src/lib/components/move-verify-section/VerifiedDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Text } from "@chakra-ui/react";
+
+export const VerifiedDetails = () => {
+  return (
```
Comment (songwongtp) [inline src/lib/components/move-verify-section/VerifiedDetails.tsx:4]: implicit return

File: src/lib/components/module/ModuleSourceCode.tsx [hunk 2]
```diff
@@ -26,36 +33,81 @@ const loadMoveSyntax = (monaco: Monaco) => {
...
+                    Verified Module Source Code
+                  </Heading>
+                </Flex>
+                <Text
+                  fontWeight={600}
+                  variant="body2"
+                  color="text.dark"
+                  textAlign="start"
+                >
+                  {/* TODO: Add timestamp */}
```
Comment (songwongtp) [inline src/lib/components/module/ModuleSourceCode.tsx:63]: fix todo?

File: src/lib/pages/module-details/components/ModuleTop.tsx [hunk 3]
```diff
@@ -208,17 +214,11 @@ export const ModuleTop = ({ moduleData, isVerified }: ModuleTopProps) => {
...
-              <Flex>
-                <CustomIcon
-                  name="check-circle-solid"
-                  boxSize={5}
-                  color="success.main"
-                />
-              </Flex>
-            </Tooltip>
-          )}
+          {/* TODO: Add status & remove isVerified */}
```
Comment (songwongtp) [inline src/lib/pages/module-details/components/ModuleTop.tsx:217]: still todo?

**Review by songwongtp — APPROVED** [review]

**Review by songwongtp — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Pull request merged**